include/dpp/utility.h

```diff
@@ -253,6 +253,13 @@ namespace dpp {
 		 */
 		bool DPP_EXPORT has_voice();
 
+		/**
+		 * @brief Returns true if D++ was built with coroutine support
+		 * 
+		 * @return bool True if coroutines are supported 
+		 */
+		bool DPP_EXPORT is_coro_enabled();
```

Comment:
why is this in this pr?
please do not combine various different features into one pr as they will be squash merged.